app/controllers/submit.js

```diff
@@ -105,6 +105,10 @@ export default Ember.Controller.extend(BasicsValidations, NodeActionsMixin, Tagg
     userNodes: Ember.A(),
     userNodesLoaded: false,
 
+    availableLicenses: Ember.A(),
+    applyLicense: false,
+    newNode: false,
```

Comment:
These three fields need to be added to the `clearFields` method which restores submitForm defaults. For example, if you submit a preprint and then hit the back button, the clearFields method makes sure you don't have lingering data.